src/main/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/movements/TemporaryAbsenceMovementMapping.kt

```diff
@@ -0,0 +1,48 @@
+package uk.gov.justice.digital.hmpps.nomisvisitsmappingservice.movements
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.data.annotation.Id
+import org.springframework.data.annotation.Transient
+import org.springframework.data.domain.Persistable
+import java.time.LocalDateTime
+import java.util.*
```

Comment:
I have noticed this being accepted more now - didn't think it was allowed, but possibly ok now.